tRecorderApi/api/tests/tests_tr.py

```diff
@@ -0,0 +1,85 @@
+        self.language_object = Language(slug='en-x-demo', name='english', id=1)
+        self.book_object = Book(name='Mark', slug='en', booknum=5, id=1)
+        self.user_object = User(name='testy', agreed=True, picture='mypic.jpg', id=1)
+        self.comment_object = Comment(location='/test-location/', id=1)
+
+    def test_that_tR_file_was_returned_in_response_from_wav_files(self):
+        """Verify that files are ready for exporting in a folder with file extension tR only"""
+        self.take_object.save()
+        self.language_object.save()
+        # from SourceFileView class
+        # upload zip maybe?
+        self.client.post(base_url + 'upload/zip', {'Media type': '*/*', 'Content': 'en-x-demo2_ulb_mrk.zip'})
+        self.response = self.client.post(base_url + 'get_source', {'language': 'en-x-demo', 'version': 'ESV', 'book': 'en'},
+                                         format='json')
+        # just checking for existence of .tr file extension
+        self.assertIn('en-x-demo_ESV.tr', os.listdir(tr_path))
+        self.take_object.delete()
+        self.language_object.delete()
+
+    def test_that_tR_is_in_correct_directory(self):
+        """Verify that tR was created in correct directory"""
+        self.book_object.save()
+        self.user_object.save()
+        self.comment_object.save()
+        self.language_object.save()
+        self.take_object.save()
+        self.response = self.client.post(base_url + 'get_source/', {'language': 'en-x-demo', 'version': 'ESV', 'book': 'en'},
+                                         format='json')
+        # double check tr_path variable
+        self.assertTrue(os.path.exists(tr_path))
+        self.take_object.delete()
+        self.language_object.delete()
+        self.book_object.delete()
+        self.user_object.delete()
+        self.comment_object.delete()
+
+
+    #Fails For Now Will Be Tested Later When We Have More Understanding of TR Files
+    # def test_that_source_audio_in_tR_file_is_in_MP3_format(self):
+    #     """Verify that source audio in tR file contains files with MP3 file ext. only"""
+    #     self.book_object.save()
+    #     self.user_object.save()
+    #     self.comment_object.save()
+    #     self.language_object.save()
+    #     self.take_object.save()
+    #     self.response = self.client.post(base_url + 'get_source/', {'language': 'en-x-demo', 'version': 'ESV', 'book': 'en'},
+    #                                      format='json')
+    #     ##fix directory location possibly if previous test doesn't work either
+    #     if any(File.endswith(".mp3") for File in os.listdir(tr_path)):
+    #         i = 1
+    #     else:
+    #         i = 0
+    #
+    #     self.assertTrue(i == 1)
+    #     # assert that media/tmp file_path_mp3 contains mp3 files
+    #     self.take_object.delete()
+    #     self.language_object.delete()
+    #     self.book_object.delete()
+    #     self.user_object.delete()
+    #     self.comment_object.delete()
```

Comment:
@Pinto18 Can you remove these block comments if the code isn't still needed for some reason in there?